backend/addcorpus/schemas/corpus.schema.json

```diff
@@ -0,0 +1,226 @@
+            "type": "string",
+            "description": "Internal name. Used in the URL and the database."
+        },
+        "meta": {
+            "type": "object",
+            "description": "Metadata about the corpus",
+            "properties": {
+                "title": {
+                    "type": "string",
+                    "description": "Human-friendly name"
+                },
+                "description": {
+                    "type": "string",
+                    "description": "Longer description"
+                },
+                "languages": {
+                    "type": "array",
+                    "description": "IETF tags of languages used in the content",
+                    "items": {
+                        "type": "string"
+                    },
+                    "uniqueItems": true,
+                    "minItems": 1
+                },
+                "category": {
+                    "type": "string",
+                    "description": "nature of the content",
+                    "enum": [
+                        "parliament",
+                        "periodical",
+                        "finance",
+                        "ruling",
+                        "review",
+                        "inscription",
+                        "oration",
+                        "book",
+                        "informative"
+                    ]
+                },
+                "date_range": {
+                    "type": "object",
+                    "description": "the date range of the content",
+                    "properties": {
+                        "min": {
+                            "type": "string",
+                            "format": "date"
+                        },
+                        "max": {
+                            "type": "string",
+                            "format": "date"
+                        }
+                    }
+                }
+            },
+            "required": [
+                "title",
+                "description",
+                "languages",
+                "category",
+                "date_range"
+            ]
+        },
+        "source_data": {
+            "type": "object",
+            "properties": {
+                "type": {
+                    "type": "string",
+                    "description": "data type of the source files",
+                    "enum": ["csv"]
+                },
+                "options": {
+                    "type": "object",
+                    "description": "additional options for source files",
+                    "properties": {
+                        "delimiter": {
+                            "type": "string",
+                            "description": "delimiter between values in the source files",
+                            "enum": [",", ";", "\t"]
+                        }
+                    }
+                }
+            }
+        },
+        "fields": {
```

Comment:
I merged these files to we just have a single `corpus.schema.json`, rather than using the URI of `field.schema.json`, because it's easier to implement than referencing a URI. Also, the resulting file isn't really that big.